packages/annotations/src/format/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import { annotation } from './annotation';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	registerFormatType,
+} from '@wordpress/rich-text';
+
+const { name, ...settings } = annotation;
+
+registerFormatType( name, settings );
```

Comment:
Not entirely related to this pull request, but in mind of #11611, there's an inconsistency we're considering in how / when formats become registered, particularly for "core" formats not specifically registered within `@wordpress/format-library`.